src/Minio.AspNetCore/MinioClientFactory.cs

```diff
@@ -20,7 +20,7 @@ public MinioClient CreateClient(string name)
     {
       var options = optionsMonitor.Get(name);
 
-      var client = new MinioClient()
+      using var client = new MinioClient()
```

Comment:
Does this using will dispose client and return disposed instance from CreateClient?

Reply:
It seems like IMinioClientFactory owner is responsible to add usings from calling side